### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by SamHPurp**
~~Removes fun~~

:cl: Purpose2
fix: Fixes assorted wording / capitalisation / grammatical problems.
fix: Fixes the exploit to produce infinite cable coil from airlocks.
/ :cl:

### CODE REVIEW

File: code/game/objects/structures/door_assembly.dm [hunk 1]
```diff
@@ -212,8 +212,9 @@
 
 		if(do_after(user, 40 * W.toolspeed, target = src))
 			if(!src) return
-			to_chat(user, "\blue You cut the airlock wires.!")
-			new/obj/item/stack/cable_coil(src.loc, 1)
+			to_chat(user, "\blue You cut the airlock wires!")
+			if(state ==1)
```
Comment (tigercat2000) [inline code/game/objects/structures/door_assembly.dm]: please keep spaces on both sides of operators

Comment (SamHPurp) [inline code/game/objects/structures/door_assembly.dm]: Sorted, thanks for reviewing.

File: code/game/gamemodes/objective.dm [hunk 2]
```diff
@@ -768,5 +768,5 @@ var/list/potential_theft_objectives = subtypesof(/datum/theft_objective) \
 //wizard
 
 /datum/objective/wizchaos
-	explanation_text = "Wreak havoc upon the station as much you can. Send those wandless Nanotrasen scum a message!"
+	explanation_text = "Wreak havoc upon the station as much you can. Send those wandless NanoTrasen scum a message!"
```
Comment (Fox-McCloud) [inline code/game/gamemodes/objective.dm]: This is incorrect. We decided a long time ago to move away from Bay's `NanoTrasen` and go with `Nanotrasen`

File: code/game/objects/items/weapons/tools.dm [hunk 3]
```diff
@@ -305,7 +305,7 @@
 //Welding Tool
 /obj/item/weapon/weldingtool
 	name = "welding tool"
-	desc = "A standard edition welder provided by Nanotrasen."
+	desc = "A standard edition welder provided by NanoTrasen."
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/tools.dm]: Nope.

File: code/game/gamemodes/objective.dm (hunk 2 above)
Comment (Regen1) [inline code/game/gamemodes/objective.dm]: When did we do that?

Comment (SamHPurp) [inline code/game/gamemodes/objective.dm]: As @Regen1 says, I'm unaware of that. If thats the case, then other places such as the wiki need to be updated.

If you could please confirm @Necaladun @FreeStylaLT on NanoTrasen vs Nanotrasen, I'll update this PR, and make sure the wiki falls in line too.

Comment (Fox-McCloud) [inline code/game/gamemodes/objective.dm]: A very long time ago: https://github.com/ParadiseSS13/Paradise/pull/3

Any instance of `NanoTrasen` is in error and should be corrected.

Comment (Regen1) [inline code/game/gamemodes/objective.dm]: Then we should probably revisit that, as it is at all not something directly "codebase related" and was a decision that mark and mel did entirely on their own.

Comment (SamHPurp) [inline code/game/gamemodes/objective.dm]: @Regen1 @Fox-McCloud I'll revert these for the time being.

If you think this conversation is worth picking up again though @Regen1 - how would you best recommend this?

**Comment by SamHPurp** [comment]
@Fox-McCloud NT capitalisation changes reverted as per our discussion.

**Pull request merged**